### PR TITLE
Build Error hotfix

### PR DESCRIPTION
**Pull request by johnbyrne727rit**
Went through the build process (including docker spinup, db migration, npm install) of both front end and back end. Got rid of any error causing issues. Some errors were from code getting blown away in a merge, rebuilt that based off of git history. Double checked that visual wonkiness / FE form misbehavior wasn't introduced.

I think in the future to avoid this, we should always
- Pull dev into our own branch prior to review requesting our prs
- Rebuild and rerun our changes after pulling dev in, prior to review requesting our prs

That way if something like this happens again it will get caught either when we rebuild and run or when everyone else reviews the pr.

### CODE REVIEW

**Review by JohnPMurray — APPROVED** [review]
Looks good. Did you run a lint or something there’s a bunch of spacing changes

**Comment by johnbyrne727rit** [comment]
I ran prettier over the sections where I had to rebuild some stuff out. I tried to do it localized to my selection, but I guess it did the whole files for those 😨 . Should be 100% cosmetic (and 100% towards my code ownership mwahaha).

File: frontend/src/Student/components/VideoSubmissionForm.js [hunk 1]
```diff
@@ -233,61 +229,62 @@ class VideoSubmissionForm extends Component {
...
-                  />
+                  {values.submittingAsGroup === "no" ? (
+                    <HomeTownInput
+                      hometownNeeded={hometownNeeded}
+                      values={values}
+                      touched={touched}
+                      errors={errors}
+                      renderErrors={this.renderErrors}
+                    />
+                  ) : null}
```
Comment (johnbyrne727rit) [inline frontend/src/Student/components/VideoSubmissionForm.js:287]: This is one of the things that got blown away

File: frontend/src/Student/components/OtherMediaSubmissionForm.js [hunk 2]
```diff
@@ -304,49 +304,50 @@ class OtherSubmissionForm extends Component {
...
-                  />
+                  {values.submittingAsGroup === "no" ? (
+                    <HomeTownInput
+                      hometownNeeded={hometownNeeded}
+                      values={values}
+                      touched={touched}
+                      errors={errors}
+                      renderErrors={this.renderErrors}
+                    />
+                  ) : null}
```
Comment (johnbyrne727rit) [inline frontend/src/Student/components/OtherMediaSubmissionForm.js:351]: This was the other spot that had a merging issue

**Comment by johnbyrne727rit** [comment]
Those two in line comments show the non/style related changes for the files that got prettified

**Review by JohnPMurray — CHANGES_REQUESTED** [review]
Screen Shot 2020-02-11 at 6 39 41 PM 1
Still broken, Im gonna take a crack at it

File: backend/resolvers/mutations/entry.js [hunk 3]
```diff
@@ -61,7 +61,7 @@ const createEntry = (entry, entryType, entryId, t) => {
           )
         }
 
-        else if (user.displayName != entry.displayName){
+        else if (!group && user.displayName != entry.displayName){
```
Comment (JohnPMurray) [inline backend/resolvers/mutations/entry.js:64]: This should ensure that it only checks for a hometown and display name on single user submissions.

**Review by JohnPMurray — APPROVED** [review]
Error is fixed now all tests pass

File: frontend/src/Student/components/PhotoSubmissionForm.js [hunk 4]
```diff
@@ -240,14 +240,10 @@ class PhotoSubmissionForm extends Component {
                       participants: values.groupParticipants
                     }
                     : null,
-<<<<<<< HEAD
```
Comment (JohnPMurray) [inline frontend/src/Student/components/PhotoSubmissionForm.js:243]: Why dont we have this conflict in other media submission?

Comment (JohnPMurray) [reply frontend/src/Student/components/PhotoSubmissionForm.js:243]: I think other media is missing the displayName field here don't we need that?

**Review by JohnPMurray — CHANGES_REQUESTED** [review]
Looks like othermediasubmission isn't passing displayName to the database. Not sure if it's the same issue, but we might as well fix it here. I'll get it.

File: frontend/src/Student/components/OtherMediaSubmissionForm.js [hunk 5]
```diff
@@ -185,117 +185,118 @@ class OtherSubmissionForm extends Component {
...
-                    }
+                        creatorUsername: user.username,
+                        participants: values.groupParticipants
+                      }
                     : null,
-                hometown: values.submittingAsGroup === 'no'?  
-                  values.hometown
-                  : null,
-                studentUsername: values.submittingAsGroup === 'no' ? user.username: null,
+                displayName: values.displayName,
```
Comment (JohnPMurray) [inline frontend/src/Student/components/OtherMediaSubmissionForm.js:237]: Just added this, I think that does it.

**Review by JohnPMurray — APPROVED** [review]
Third time's the charm. I think we're good here

File: frontend/src/Student/components/PhotoSubmissionForm.js (hunk 4 above)
Comment (johnbyrne727rit) [reply frontend/src/Student/components/PhotoSubmissionForm.js:243]: I think `displayName` popped in above the `hometown` field in this file, so the delete conflict didn't pop in.

**Review by johnbyrne727rit — COMMENTED** [review]
@JohnPMurray Your changes look good to me.

**Review by gollb310 — APPROVED** [review]
Looks good.

**Pull request merged**